pymongo/client_session.py

```diff
@@ -986,7 +986,7 @@ def _apply_to(
         self,
         command: MutableMapping[str, Any],
         is_retryable: bool,
-        read_preference: _ServerMode,
+        read_preference: Optional[_ServerMode],
```

Comment:
This is unexpected, when is read_preference None?

Reply:
I still see the Optional here.

Reply:
hm wacky..i agree that in this comment it looks like its still Optional here, but when I go to the files changed tab, it doesn't look like its there anymore? I'm confused because did commit and push those changes.